### ISSUE TITLE
Adapting pyCycle to extra-terrestrial atmosphere

### ISSUE THREAD

**Issue by alice4space** [OpenMDAO/pyCycle#50]
Is it possible to adapt pyCycle for it to work in Martian atmosphere for example ?

**Comment by eshendricks** [comment]
Yes, it should be possible to adapt pyCycle to model something operating in a Martian atmosphere.  Off the top of my head, you would need to specify the proper properties of the atmosphere (i.e. composition) and the fuel in the thermodynamics package (either through running CEA or inputing a table of properties).  Also, the US standard atmosphere is used to determine pressure and temperature as a function of altitude in the flight conditions group, so component likely would need to be replaced  as well.
